examples/evaluate_multiple_isolated_robots/main.py

```diff
@@ -0,0 +1,73 @@
+"""
+Simulate modular robot that do not interact, and then calculate their xy displacement.
+
+To understand better, first see the 'evaluate_single_robot' example.
+
+You learn:
+- How to simulate multiple isolated robots.
+- How to process their simulation results.
+"""
+
+import asyncio
+
+from revolve2.core.modular_robot import (
+    ModularRobot,
+    get_body_states_multiple_isolated_robots,
+)
+from revolve2.core.modular_robot.brains import BrainCpgNetworkNeighborRandom
+from revolve2.runners.mujoco import LocalRunner
+from revolve2.standard_resources import fitness_functions, modular_robots
+from revolve2.standard_resources import terrains as standard_terrains
+from revolve2.standard_resources.rng import make_rng
+from revolve2.standard_resources.simulation import (
+    create_batch_multiple_isolated_robots_standard,
+)
+
+
+def main() -> None:
+    """Run the simulation."""
```

Comment:
missing standard logging setup here?

Reply:
It's not really needed there but to not have students copy paste code and forget to initialize logging I added it to all examples.